### PR TITLE
Add db2data container to docker-compose

### PR DESCRIPTION
**Pull request by denys-davydov**
## Description
Added db2 container to docker-compose to support the latest version of cans-api.
**N.B.** Updated .env for local usage can be found here: https://github.com/ca-cwds/env-store/blob/master/envs/cans/.env

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: docker-compose.ci.yml [hunk 1]
```diff
@@ -52,6 +63,14 @@ services:
...
       DB_NS_USER: ${DB_NS_USER}
       DB_NS_PASSWORD: ${DB_NS_PASSWORD}
+      DB_CMS_JDBC_URL: ${DB_CMS_JDBC_URL}
+      DB_CMS_SCHEMA: ${DB_CMS_SCHEMA}
+      DB_CMS_USER: ${DB_CMS_USER}
+      DB_CMS_PASSWORD: ${DB_CMS_PASSWORD}
+      DB_CMS_RS_JDBC_URL: ${DB_CMS_RS_JDBC_URL}
+      DB_CMS_RS_SCHEMA: ${DB_CMS_RS_SCHEMA}
+      DB_CMS_RS_USER: ${DB_CMS_RS_USER}
+      DB_CMS_RS_PASSWORD: ${DB_CMS_RS_PASSWORD}
```
Comment (igor-chornobay-engagepoint) [inline docker-compose.ci.yml]: Why do we need CMS_RS schema? This is the replicated schema used by Jobs (Neutron) to populate ES Index/

File: docker-compose.ci.yml [hunk 2]
```diff
@@ -1,7 +1,17 @@
 version: '3.2'
 services:
+  db2:
+    image: cwds/db2data
+    hostname: db2data
```
Comment (igor-chornobay-engagepoint) [inline docker-compose.ci.yml]: I suggest we specify particular tag here.

File: docker-compose.ci.yml (hunk 1 above)
Comment (LeonidMarushevskyi) [reply docker-compose.ci.yml]: For client county information.

Comment (igor-chornobay-engagepoint) [reply docker-compose.ci.yml]: ok

Comment (denys-davydov) [reply docker-compose.ci.yml]: @igor-chornobay-engagepoint Here is the story for it https://osi-cwds.atlassian.net/browse/CANS-578 
Sill not sure it answers your question, but Leonids answer surely does.

File: docker-compose.ci.yml (hunk 2 above)
Comment (denys-davydov) [reply docker-compose.ci.yml]: Unless we have an active development of db2data image and regular updates in the image I would prefer to have the _latest_ tag.

Comment (igor-chornobay-engagepoint) [reply docker-compose.ci.yml]: ok

**Pull request merged**